desk/app/expose/page.hoon

```diff
@@ -10,12 +10,15 @@
 ++  render
   |=  [=bowl:gall =nest:g:c msg=post:d]
   ^-  (unit manx)
+  =/  author=@p
+    ?@  author.msg  author.msg
+    ship.author.msg
   =/  aco=(unit contact-0:co)
-    (get-contact:co bowl author.msg)
+    (get-author-contact:r bowl author.msg)
   ::
   ::TODO  if we render replies then we can "unroll" whole chat threads too (:
-  |^  ?-  -.kind-data.msg
-          %chat
+  |^  ?+  kind.msg  ~
+          [%chat ~]
```

Comment:
This is maybe getting ahead of ourselves, but the way kind is gonna be used at the client side is by extending the basic path. Those extended post types would not be rendered by %expose, which we should keep in mind.

@latter-bolden @davidisaaclee

Reply:
Do any of you think it'd make sense to render the extended post types as their normal post types for now, given that the alternative is not rendering them at all?

(Not that it matters much, it's "just" expose, after all.)

Reply:
It's fine for expose to drop client customizations for now. We can address later when we're "productionizing" public profiles.